packages/spartan/extra/gen_resolvconf.py

```diff
@@ -51,6 +51,7 @@ def check_server(addr):
 
 options timeout:1
 options attempts:3
+options edns0
```

Comment:
Whats the worst case when they dont support edns0? Would we need to check for this support and make sure it works before anyone uses this? Doc change?